src/index.html

```diff
@@ -6,9 +6,29 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="./images/logo.png" alt="MOYO Logo"></a>
+      <nav class="nav">
+        <ul class="categories">
+          <li class="button"><a class="link is-active" href="#">Apple</a></li>
+          <li class="button"><a class="link" href="#">Samsung</a></li>
+          <li class="button"><a class="link" href="#">Smartphones</a></li>
+          <li class="button"> <a class="link" data-qa="hover"
+                href="#">Laptops & Computers</a>
+          </li>
+          <li class="button"><a class="link" href="#">Gadgets</a></li>
+          <li class="button"><a class="link" href="#">Tablets</a></li>
```

Comment:
```suggestion
          <li class="button">
            <a class="link" href="#">
              Gadgets
            </a>
          </li>
       
          <li class="button">
            <a class="link" href="#">
              Tablets
            </a>
          </li>
```

Reply:
<img width="975" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/adffb000-eb06-47a2-84bd-c27d1bc65bdf">
Fix in all places